File: projects/jam/jam_postcompute.lua
function jam_postcompute(project)
	print('POST COMPUTE:')
	print(project:getTag('return_temp'))
	project:setTag('return_temp',50)

	local unit = project:getUnit('CipUnit')
	print(unit.name)

	local stream = project:getStream('fresh_water')
	print(stream.name, stream.tin, project:getTag(stream.tin))
end

File: projects/jam/jam_postcompute.m
disp('Run in Matlab');

u = getUnit('CipUnit');
s = getStream('fresh_water');
t = getTag('return_temp');
v = setTag('return_temp', 150);

disp(['Unit Name: ', u.name]);
disp(['Stream Name: ', s.name]);
disp(['Tag Value: ', (t)]);
disp(['New Tag Value: ', (v(1,1))]);
disp(['Check Tag Value: ', (getTag('return_temp'))]);

clear all;